src/utilities/sanitize-value.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Sanitizes the input `value` based on the specified `type`.
+ * Validates that the `value` is of the correct type and that `type` is a valid string.
+ * If any validation fails, logs an error message and returns `null`.
+ *
+ * @param {*} value - The value to be validated. Can be of various types including boolean, string, array, object, number, HTMLElement, bigint, symbol, or function.
+ * @param {string} type - The expected type of the `value`. Should be a string representing the type (e.g., "boolean", "string", "array", "object", "number", "HTMLElement", "bigint", "symbol", "function", or "null").
+ *
+ * @returns {*} The original `value` if all validations pass; otherwise, `null`.
+ */
+function sanitizeValue(value, type) {
+	let flag = false
```

Comment:
We just talked about this before. @coderabbitai, a flag is helpful because I don't want to return immediately.

Reply:
@coderabbitai, using a flag in the `sanitizeValue` function is helpful because it keeps error handling organized. ✅ The flag allows to track whether any of the multiple validation checks fail, so you can perform all checks before deciding to return null, rather than stopping at the first error. This makes the code cleaner and easier to understand by clearly showing that you're running several checks and only taking action if needed. 🛠️ This approach helps you handle multiple potential issues in a single, manageable way. 🚀